Source/Core/BaseRow.swift

```diff
@@ -212,6 +212,11 @@ extension BaseRow {
         removeFromRowObservers()
     }
 
+    final func willBeRemovedFromSection() {
```

Comment:
@mtnbarreto my thought when i did this change: if you are removing a row from a section then it will be removed from both the form and the section. But if you are removing a section from a form, then its rows will only be removed from the form but they will stay in the section. Let me know what do you think